### ISSUE TITLE
new traffic profile reset all values

### ISSUE THREAD

**Issue by svenvogel** [ntop/ntopng#395]
Hi,

when i add a new traffic profile my complete other profiles will be reset and cleared. why is that so? its not good because we need the traffic from the days before for complete month.

thanks

Sven

**Comment by simonemainardi** [comment]
@svenvogel, please post the exact steps that lead to the issue you are experiencing, thank you. I'm trying to reproduce it with no luck.

**Comment by svenvogel** [comment]
Hi Simone,

before it looks with my values over any days...interfaces - my probe - overview
![beforce_traffic](https://cloud.githubusercontent.com/assets/5433844/13094626/315429ba-d50c-11e5-8b6b-3047a26a639f.PNG)
1. got to traffic profiles
2. change any traffic profile and click save
3. go back to interfaces and my nprobe and choose the overview again and counter starts new...

![after_traffic](https://cloud.githubusercontent.com/assets/5433844/13094644/522d95a4-d50c-11e5-9d82-f65a44fe0dae.PNG)

**Comment by lucaderi** [comment]
This is the way it is designed. Are you saying that we should restart just one counter (the one you modified)?

**Comment by svenvogel** [comment]
Hi Luca,

yes we think that would be better. first we thought it is the complete count of one traffic profile. we found out that we found the complete traffic director in the profile. 

![image](https://cloud.githubusercontent.com/assets/5433844/13149259/27b59c30-d661-11e5-8503-4ce570a0c76a.png)

![image](https://cloud.githubusercontent.com/assets/5433844/13149267/30254fdc-d661-11e5-9863-d1c5bd72894f.png)

We hope you will add a report system where we can choss not only 1m 5m 10m and so on where we can choose from 01.01 to 31.01 and get the complete traffic for profile and pdf export ;).

Sven
